boards/msbiot/doc.txt

```diff
@@ -12,130 +12,105 @@ The MSB-IoT was developed at FU Berlin and is based on a STM32F415RG MCU.
+| Programming Manual    | [Programming Manual](http://www.st.com/st-web-ui/static/active/en/resource/technical/document/programming_manual/DM00046982.pdf)  |
+
+
+### GPIO Pins
 
 The following picture shows the easily accessible pins of the board:
-![MSB-IoT Pinout](https://raw.githubusercontent.com/wiki/RIOT-OS/RIOT/images/msbiot-pinout.png)
 
-RIOT uses its own naming scheme for GPIO pins and other functionality. The
-actual STM32F4 specific pins and functional units are mapped to RIOT internal
-enum values. Please refer to [this document](https://docs.google.com/spreadsheets/d/1RLn1j5h7VIQ1zezRs5Z3A7SYCMvEukHaKjhcsyrDz6g/pubhtml)
-for RIOTs pin mapping
-for the MSB-IoT (the document only lists ports where a PIN is mapped).
-
-The pin mapping can be altered freely by editing the
-`boards/msbiot/include/periph_conf.h` header file found
-[here](https://github.com/RIOT-OS/RIOT/tree/master/boards/msbiot/include).
-Please keep in mind that changing pin mappings that are important for the
-integrated hardware could affect the corresponding driver implementations. The
-most important mappings needed for correct operation of the MSB-IoT with RIOT
-can be extracted from the `boards/msbiot/include/board.h` found
-[here](https://github.com/RIOT-OS/RIOT/tree/master/boards/msbiot/include).
+![MSB-IoT Pinout](https://raw.githubusercontent.com/wiki/RIOT-OS/RIOT/images/msbiot-pinout.png)
 
 ### User Interface
 
-**3 Buttons:**
+#### 3 Buttons
+
+| NAME  | User Button T1    | User Button T2    | T RESET   |
+|:----- |:----------------- |:----------------- |:--------- |
+| Pin   | PB13              | PA0               | NRST      |
 
-| NAME   | User Button T1    | User Button T2    | T RESET|
-|:----- |:----- |:--------- |:------|
-| Pin    | PB13 | PA0    | NRST|
 The user buttons are mapped to GPIO_0 (Button T2) and GPIO_8 (Button T1) in
 the board's default configuration file for RIOT. To configure and use the
-buttons, you can use the provided interface functions from RIOT's
-[GPIO](https://github.com/RIOT-OS/RIOT/blob/master/drivers/include/periph/gpio.h) low-level driver interface.
+buttons, you can use RIOT's @ref drivers_periph_gpio.
 
 Pushing the reset button will always reset the board and restart your
 application.
 
-**3 user controllable LEDs:**
+#### 3 user controllable LEDs
+
+| NAME  | LED_RED   | LED_YELLOW    | LED_GREEN |
+| ----- | --------- | ------------- | --------- |
+| Color | red       | yellow        | green     |
+| Pin   | PB8       | PB14          | PB15      |
 
-| NAME  | LED_RED   | LED_YELLOW | LED_GREEN |
-| ----- | --------- | ---------- | --------- |
-| Color | red       | yellow     | green     |
-| Pin   | PB8       | PB14       | PB15      |
 The LEDs can be controlled inside your RIOT application by using the LED
-macros provided [here](https://github.com/RIOT-OS/RIOT/blob/master/boards/msbiot/include/board.h).
+macros provided in @ref boards_msbiot.
```

Comment:
Using this `@ref` is self referencing the board documentation page. I see 2 other possibilities:
- reference the board.h file (the link is http://doc.riot-os.org/msbiot_2include_2board_8h.html)
- directly give the macro names here.

Reply:
Thanks for spotting that. I directly referenced the macro names now.